openedx/core/djangoapps/schedules/management/commands/tests/test_send_upgrade_reminder.py

```diff
@@ -74,20 +92,27 @@ def test_schedule_bin(self, schedule_count, mock_schedule_send, mock_ace):
         schedules = [
             ScheduleFactory.create(
                 upgrade_deadline=datetime.datetime(2017, 8, 3, 18, 44, 30, tzinfo=pytz.UTC),
-                enrollment__user=UserFactory.create(),
+                enrollment__user=UserFactory.create(id=(i + 100)),
```

Comment:
Why are we adding 100 here? Django should be giving us unique ids by default...

Reply:
In fact, why are we specifically calling `UserFactory.create()` at all? The `CourseEnrollment` factory should be creating it for us.

Reply:
I think we can get rid of this, at one point I was trying to control the buckets that users ended up in, but that's no longer necessary.